test/build/base-url-homepage/base-url-homepage.test.js

```diff
@@ -1,18 +1,26 @@
-const fs = require('fs');
 const path = require('path');
 const cheerio = require('cheerio');
+const {setupBuildTest, readFiles} = require('../../test-utils');
 
-const html = fs.readFileSync(path.join(__dirname, 'build', 'index.html'), 'utf8');
+const cwd = path.join(__dirname, 'build');
 
-const $ = cheerio.load(html);
+let files = {};
 
 describe('packageManifest.homepage', () => {
+  beforeAll(() => {
+    setupBuildTest(__dirname);
```

Comment:
This one line does have to be added to the beginning of every test, however, it can be omitted if a different setup is required. Hopefully it‘s the best of both worlds! And it keeps the functionality where if you only want to run a single test locally, only that one test will build.

Reply:
Yea I like this! However, using the synchronous `execSync` here probably slows these tests down a ton more than needed. To do this, Node/execa has to block the thread until the command continues. This prevents running multiple tests in parallel, unless Node is spinning up workers for each test file. 

In either case, you can return a promise here, so strong +1 for using the async `execa` command and then awaiting that here, so that Jest can go do other stuff while the test folder sets up

Reply:
That‘s a good point about running in parallel. I usually try not to pre-emptively optimize something but it does make more sense to flow with how the tests are running. Will change